packages/vendure-plugin-invoices/src/services/invoice.service.ts

```diff
@@ -127,12 +129,13 @@ export class InvoiceService implements OnModuleInit, OnApplicationBootstrap {
    * Listen for OrderPlacedEvents. When an event occures, place generate-invoice job in queue
    */
   onApplicationBootstrap(): void {
-    this.eventBus.ofType(OrderPlacedEvent).subscribe(async ({ ctx, order }) => {
-      this.createInvoiceGenerationJobs(ctx, order.code, 'order-placed');
+    this.eventBus.ofType(OrderPlacedEvent).subscribe(({ ctx, order }) => {
+      void this.createInvoiceGenerationJobs(ctx, order.code, 'order-placed');
```

Comment:
Same for this one, should also have a catch.

Good for finding this one, because the previous implementation also had the possibility to crash the node server!